docs/fidesops/docs/guides/policies.md

```diff
@@ -107,21 +107,27 @@ To configure a storage destination, see [How-To: Configure Storage](storage.md)
   },
 ]
 ```
+This will create a rule to hash a not-yet-specified value with a SHA-512 hash. We need to specify a target to hash, so we need to creat a target for this rule:
 
-`PUT api/v1/policy/a-demo-policy/rule/mask-provided-emails`
-
-```json 
-[
-  {
-    "data_category": "user.provided.identifiable.contact.email",
-  },
-]
+```json
+PUT api/v1/policy/{policy_key}/rule/{rule_key}
+  [
+    {
+      "data_category": "user.provided.identifiable.contact.email",
+    },
+  ]
 ```
 
-The `Policy` specified above will now:
-
+This policy, `user_email_address_polcy`, will now do the following:
 - Return all data inside the Identity Graph with a data category that matches (or is nested under) `user.provided.identifiable.contact`.
 - Mask all data inside the Identity Graph with a data category that matches `user.provided.identifiable.contact.email` with a the `SHA-512` hashing function.
 
+### A Note About Erasing Data
+
+When a Policy Rule erases data, it erases the _entire_ branch given by the Target. For example, if we created an `erasure` rule with a Target of `user.provided.identifiable.contact`, _all_ of the information within the `contact` node -- including `email` -- would be erased.
```

Comment:
> _all_ of the information within the `contact` node -- including `email` -- would be erased

This could confuse people, because until you read the next line it's not clear that `email` isn't under the `user.provided.identifiable.contact...` branch